packages/api/src/controllers/reviews/v1/mod.rs

```diff
@@ -32,19 +34,28 @@ impl ReviewControllerV1 {
         State(ctrl): State<ReviewControllerV1>,
         Path(id): Path<i64>,
     ) -> Result<Json<Review>> {
-        //TODO: implement get review
         let _repo = ctrl.review_repo;
         tracing::debug!("get_review: {:?}", id);
 
-        Ok(Json(Review::default()))
+        let fetched_review = sqlx::query_as!(
+            Review,
+            "SELECT * FROM reviews WHERE id = $1",
+            id
+        )
```

Comment:
```suggestion
        self.review_repo.delete(id)
```
It's enough to delete Review model.